tests/sentry/utils/sdk_crashes/test_sdk_crash_detection.py

```diff
@@ -60,6 +69,9 @@ class CococaSDKTestMixin(BaseSDKCrashDetectionMixin):
     def test_unhandled_is_detected(self, mock_sdk_crash_reporter):
         self.execute_test(get_crash_event(), True, mock_sdk_crash_reporter)
 
+    def test_feature_disabled_unhandled_is_not_detected(self, mock_sdk_crash_reporter):
+        self.execute_test(get_crash_event(), False, mock_sdk_crash_reporter, feature_enabled=False)
```

Comment:
Instead of passing `feature_enabled` as an argument, I would set it as a decorator. That way, you can also get rid of the `_execute_test` helper in `execute_test`.

```suggestion
    @with_feature("organizations:sdk-crash-reporting")
    def test_feature_disabled_unhandled_is_not_detected(self, mock_sdk_crash_reporter):
        self.execute_test(get_crash_event(), False, mock_sdk_crash_reporter)
```

Reply:
OK sure, if you think that's too verbose. Another way to simplify `execute_test` is too put the entire test in a `with Feature({"organizations:sdk-crash-reporting": feature_enabled}):` block, that also makes `_execute_test` unnecessary.

Reply:
Nice, thanks @jjbayer 👍